examples/comfyui/node_pack/nodes_dynamo.py

```
@@ -0,0 +1,583 @@
+
+def _wrap_video_for_comfy(path: str):
+    """Wrap an MP4 path in ComfyUI's native VIDEO type if available; else return the path string.
+
+    Stock ComfyUI exposes ``comfy_api.latest.InputImpl.VideoFromFile``. Older builds may
+    not have it, in which case downstream nodes can still consume a path string via
+    VHS / SaveVideo.
+    """
+    try:
+        from comfy_api.latest import InputImpl  # type: ignore
+        return InputImpl.VideoFromFile(path)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)